services/eth_gateway.go

```diff
@@ -99,30 +101,76 @@ type TransactionWithBlockNumber struct {
 	Confirmed   *bool
 }
 
+// Gas Calculation for Sending Ether
 type CalculateGasToSend func(desiredGasLimit uint64) (*big.Int, error)
+
+// Generate Valid Ethereum Network Address
 type GenerateEthAddr func() (addr common.Address, privateKey string, err error)
+
+// Generate Private Keys W/O Address
 type GenerateKeys func(int) (privateKeys []string, err error)
+
+// Generate Ethereum Address from Private Keys
 type GenerateEthAddrFromPrivateKey func(privateKey string) (addr common.Address)
+
+// Generate Ethereum Public Key from Private Key
 type GeneratePublicKeyFromPrivateKey func(c elliptic.Curve, k *big.Int) *ecdsa.PrivateKey
+
+// Gas Price On Ethereum Network
 type GetGasPrice func() (*big.Int, error)
+
+// Wait For Transfer to Complete
 type WaitForTransfer func(brokerAddr common.Address, transferType string) (*big.Int, error)
+
+// Check Ethereum Balance
 type CheckETHBalance func(common.Address) /*In Wei Unit*/ *big.Int
+
+// Check PRL Balance
 type CheckPRLBalance func(common.Address) /*In Wei Unit*/ *big.Int
+
+// Get Current(Latest) Block from Ethereum Network
 type GetCurrentBlock func() (*types.Block, error)
-type SendETH func(fromAddr common.Address, fromPrivKey *ecdsa.PrivateKey, toAddr common.Address, amount *big.Int) (types.Transactions, string, int64, error)
+
+// Send Ether To Valid Ethereum Network Address
+type SendETH func(toAddr common.Address, amount *big.Int) (types.Transactions, string, int64, error)
```

Comment:
We should change this type to what's in master.  There is code elsewhere that depends on it.